SETUP.md

```diff
@@ -0,0 +1,46 @@
+# Introduction
+
+This repo serves two purposes:
+
+- A top level NPM modules that aggregates the components that makeup the Salesforce DX product
+
+This repository can be used to install different versions of Salesforce DX. Our development process maintains a number of NPM release channels which are:
+
+- master - This represents the current G/A version of Salesforce DX
+- release - This represents the current patched version of G/A version Salesforce DX
+- pre-release - This is next major version of Salesforce DX
+
+The master and release channels are published to the Heroku NPM registry. Our public customers will install Salesforce DX from that location.
+The pre-release channel is only available to internal customers, via an internal NPM registry, and this channel is only available between GS0 and R2 releases of the core app server.
+
+The master and release channels are also available from the internal NPM registry. Making these all channels avalable internally eases the burden of testing a release before it is published to the public Heroku registry.
+
+## Installing the Salesforce DX as a Customer
+
+### To install the G/A version of Salesforce DX
+
+1. Ensure that the sfdx cli is installed.
+1. `sfdx plugins:install salesforcedx`
+
+### To install a "tagged" version of Salesforce DX
+
+A customer would use this to install a previous version of Salesforce DX or install a version that has not been promoted to the "latest" version.
+
+1. Ensure that the sfdx cli is installed.
+1. `sfdx plugins:install salesforcedx@39.0.1`
+
+### To install Salesforce DX from the internal NPM registry
```

Comment:
This is internal information only, and no longer relevant. Please remove and make sure you **SQUASH** and merge this change to master before when you merge to main.